### PR TITLE
Update peer,rest,rpc Oraichain provided by Vchain

### PR DESCRIPTION
**Pull request by thanhnv2303**

### CODE REVIEW

File: oraichain/chain.json [hunk 1]
```diff
@@ -92,6 +92,11 @@
         "id": "d091cabe3584cb32043cc0c9199b0c7a5b68ddcb",
         "address": "seed.orai.synergynodes.com:26656",
         "provider": "synergynodes"
+      },
+      {
+      "id:"8b346750e75fd584645192a65c62c7ab88741791",
```
Comment (JeremyParish69) [inline oraichain/chain.json:97]: missing the " quote to close the property name "id"
should be "id": "8b...91",

Comment (thanhnv2303) [reply oraichain/chain.json]: I have fixed this issue, please recheck it.

Comment (JeremyParish69) [reply oraichain/chain.json]: sorry, it didn't work. You added a space, but not the needed missing quote

Comment (thanhnv2303) [reply oraichain/chain.json]: Sorry about my mistake. Please recheck.

**Pull request merged**